cmd/radikron-gui/frontend/src/components/WeeklyProgramBrowser.tsx

```
@@ -63,103 +181,233 @@ export const WeeklyProgramBrowser: React.FC = () => {
+                  <div>
+                    <p className="text-sm font-medium text-muted-foreground">End Time</p>
+                    <p className="text-sm text-foreground">{formatDateTime(selectedProgram.To)}</p>
+                  </div>
+                </div>
+
+                {selectedProgram.Desc && (
+                  <div>
+                    <p className="text-sm font-medium text-muted-foreground mb-1">Description</p>
+                    <div
+                      className="text-sm text-foreground [&_*]:text-foreground [&_a]:text-primary [&_a]:underline [&_a:hover]:opacity-80"
+                      dangerouslySetInnerHTML={{ __html: selectedProgram.Desc }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)